src/render.rs

```diff
@@ -164,14 +161,14 @@ struct SetGridViewBindGroup<const I: usize>;
 
 impl<const I: usize, P: PhaseItem> RenderCommand<P> for SetGridViewBindGroup<I> {
     type Param = ();
-    type ViewQuery = (Read<GridViewUniformOffset>, Read<GridViewBindGroup>);
+    type ViewQuery = (&'static GridViewUniformOffset, &'static GridViewBindGroup);
```

Comment:
Why was this changed from `Read` to `&'static`? AFAIK `Read<T>` is still valid here?

Reply:
> In Bevy 0.17, the lifetimeless module (bevy::ecs::system::lifetimeless) and its types were removed

No it wasn't? It's still there on `main` and in 0.17. https://github.com/bevyengine/bevy/blob/main/crates/bevy_ecs/src/system/system_param.rs#L2116

It's possible there's a missing feature flag or the path has changed a bit but I'd prefer if these were still used.